ch_pipeline/processing/daily.py

```diff
@@ -479,29 +617,18 @@ def csds_in_range(start, end):
     end : str or parseable to datetime
         End of interval. If `None` use now. Note that for CSD intervals the
         end is *inclusive* (unlike a `range`).
+    step : int
+        Incrementation. Default is daily.
 
     Returns
     -------
     csds : list of ints
     """
-
-    import math
-    from ch_util import ephemeris
-
     if end is None:
         end = datetime.datetime.utcnow()
 
-    if start.startswith("CSD"):
-        start_csd = int(start[3:])
-    else:
-        start_csd = ephemeris.unix_to_csd(ephemeris.ensure_unix(start))
-        start_csd = math.floor(start_csd)
-
-    if end.startswith("CSD"):
-        end_csd = int(end[3:])
-    else:
-        end_csd = ephemeris.unix_to_csd(ephemeris.ensure_unix(end))
-        end_csd = math.ceil(end_csd)
+    start_csd = datetime_to_csd(start)
+    end_csd = datetime_to_csd(end)
```

Comment:
When you factored this out, you've implicitly changed the math.ceil for end_csd to a math.floor. That might be okay, but I'm not sure.

Reply:
Oh, I did not notice that! It makes sense why it had not been factored out then. I will remove that commit for now.